charts/jaeger/Chart.yaml

```diff
@@ -30,7 +30,7 @@ dependencies:
     repository: https://charts.helm.sh/incubator
     condition: provisionDataStore.cassandra
   - name: elasticsearch
-    version: ^7.5.1
+    version: ^7.11.1
```

Comment:
@naseemkullah I feel this change would not do anything i feel. 
^7.5.1 means `>= 7.5.1 and <= 8.0.0`
^7.11.1 means `>= 7.5.1 and <= 8.0.0` as well.
https://helm.sh/docs/chart_template_guide/function_list/#caret-range-comparisons-major
or am i missing something here ? (just curious)

Reply:
Does not do much, but it would just upgrade current ESes from <7.5.1 to 7.11.1+